packages/iobio-react-components/src/percent_box.tsx

```diff
@@ -19,8 +19,30 @@
  *
  */
 
-function IobioPercentBox({ percentKey, title, totalKey }: { percentKey: string; title?: string; totalKey: string }) {
-	return <iobio-percent-box percent-key={percentKey} title={title} total-key={totalKey} />;
+import { useEffect } from 'react';
+import { setElementStyles } from './utils';
+
+function IobioPercentBox({
+	percentKey,
+	label,
+	styles,
+	totalKey,
+}: {
+	percentKey: string;
+	label?: string;
+	styles?: string;
+	totalKey: string;
+}) {
+	useEffect(() => {
+		const selector = `iobio-percent-box[percent-key=${percentKey}][total-key=${totalKey}]`;
+		const element = document.querySelector(selector);
+
+		if (element && styles) {
+			setElementStyles(element, styles);
+		}
```

Comment:
Awesome! 
PS: Took the liberty to modify your comment to replace the ZenHub link with a GitHub, so external collaborators and integrators can follow progress. (they don't have acccess otherwise)